src/main.ts

```diff
@@ -134,9 +163,12 @@ const gameLoop = (currentTime: number) => {
     // update game objects
     hummingbird && hummingbird.update(frameDelta)
     pipeController && pipeController.update(frameDelta)
+    incrementPoints()
+    showPointCounter()
```

Comment:
I hate to repeat logic but the way canvas works is it draws on top of each other, and where this is it will draw the pipes and bird over the score. I think we should move this down to just below `hummingbird && hummingbird.draw()`

making it like:
```
pipeController && pipeController.draw()
hummingbird && hummingbird.draw()
!gameOver && gameStarted && showPointCounter()
```

Reply:
Ah I was wondering why the points were behind the pipes/pencils!